example/index.ts

```diff
@@ -1,55 +1,37 @@
-import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
-
-interface UseGravityArgs {
-  moverMass: number;
-  attractorMass: number;
-  attractor: Vector<number>;
-}
-
-const applyForceForStep = (config: UseGravityArgs, mover: Entity) => {
-  const force = forceV({
-    ...config,
-    mover: mover.position
-  });
-
-  return applyForce({
-    force,
-    entity: {
-      mass: config.moverMass,
-      acceleration: mover.acceleration,
-      velocity: mover.velocity,
-      position: mover.position
+import { gravity, vector } from "../src";
+
+const el = document.createElement("div");
+document.body.appendChild(el);
+
+const positionAttractor = [500, 500] as vector<number>;
+const attractorEl = document.createElement("div");
+document.body.appendChild(attractorEl);
+
+el.style.height = "10px";
+el.style.width = "10px";
+el.style.borderRadius = "50%";
+el.style.background = "steelblue";
+
+attractorEl.style.height = "25px";
+attractorEl.style.width = "25px";
+attractorEl.style.borderRadius = "50%";
+attractorEl.style.background = "orange";
+attractorEl.style.left = `${positionAttractor[0]}px`;
+attractorEl.style.top = `${positionAttractor[1]}px`;
+attractorEl.style.position = "absolute";
+
+const [stop] = gravity({
+  config: {
+    moverMass: 2000,
+    attractorMass: 100000000,
+    attractor: positionAttractor,
+    initialVelocity: [0.01, 0.05],
```

Comment:
I added an `initialVelocity` parameter as well – just a fun way to set our `mover` on some course before the force of gravity sucks it inward. This is really fun to play around with.